pkg/abi/linux/fuse.go

```diff
@@ -517,6 +517,43 @@ type FUSEReleaseIn struct {
 	LockOwner uint64
 }
 
+// FUSECreateMeta contains all the static fields of FUSECreateIn,
+// which is used for FUSE_CREATE.
+//
+// +marshal
+type FUSECreateMeta struct {
+	Flags uint32
```

Comment:
Should we add comment to these field? just to keep it consistent with other parts